src/_footer.html

```diff
@@ -14,11 +14,13 @@ <h4 class="title"> Community </h4>
                 <a href="/community/resources.html"> Resources </a> <br />
                 <a href="/blog"> Blog </a> <br />
             </div>
-            <div class="column is-2">
+            <div class="column is-3">
                 <h4 class="title"> About </h4>
                 <a href="/about/team.html">Team</a> <br />
                 <a href="/about/advisory-board.html">Advisors</a> <br />
                 <a href="/about/jobs">Jobs</a> <br />
+                <a href="https://policies.stenci.la/privacy">Privacy Policy</a> <br />
+                <a href="https://policies.stenci.la/terms-and-conditions">Terms &amp; Conditions</a> <br />
```

Comment:
Should be just `terms` as per https://github.com/stencila/policies/pull/2/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R12

```suggestion
                <a href="https://policies.stenci.la/terms">Terms &amp; Conditions</a> <br />
```